demoscene/utils/groklinks.py

```diff
@@ -1227,6 +1227,30 @@ class HearthisTrack(AbstractBaseUrl):
     ]
 
 
+c64ch = Site("C64.CH", url='https://c64.ch/', allowed_hostnames=['c64.ch', 'www.c64.ch'],
+             icon_path='images/icons/external_sites/c64ch.png')
+
+
+class C64chProduction(UrlPattern):
+    site = c64ch
+    pattern = '/productions/<slug>'
```

Comment:
Given that we're intending to match the ID number here (because as you found, the slug is not needed), I think this should be `'/productions/<int>'`, right? (And similarly for the other patterns.)